guide/english/certifications/javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-with-the-rest-operator-to-reassign-array-elements/index.md

```diff
@@ -42,28 +41,35 @@ function removeFirstTwo(list) {
 }
 ```
 
-## Hint 3 
 
-Exclude the first two elements of the `arr` array with `,,`.
+## Spoiler Alert - Solution Ahead!
+You can use random variables to omit the first two values:
 
 ```javascript
 function removeFirstTwo(list) {
   "use strict";
   // change code below this line
-  const [,,...arr] = list; // change this
+  const [a, b, ...arr] = list; 
```

Comment:
```suggestion
  const source = [1,2,3,4,5,6,7,8,9,10];
  const [a, b, ...arr] = list; 
```
This needs to be changes since if the user copy pasted only what is currently the solution, the tests would fail

Reply:
Well I'm not sure if I've been doing it wrong but I put in the guides the part of the solution that corresponds to the "change this code" lines and maybe a bit more, but not the entire code included in the lesson to avoid clutter (already seen code) and to make solutions more to the point. Should we include a const that already exists outside the function?

Reply:
We want the entire working solution (original seed code plus any modifications to pass the challenge.

Reply:
The idea is a user should be able to copy/paste entire code shown and replace all the code in the FCC editor and pass the challenge.

Reply:
Ok, thank you.